expression_prediction_evaluation.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<p>Are you using small `l? for LFC? It's LFC not IFC. It's hard to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>